packages/core/src/plugin/type-gql-plugin-test/entitiy/node.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Field, ID as GID, ObjectType } from '@nestjs/graphql';
+import { DeepPartial, ID } from '@vendure/common/lib/shared-types';
+
+import { VendureEntity } from '../../../entity/base/base.entity';
+
+@ObjectType()
+export class BaseNode extends VendureEntity {
+    constructor(input: DeepPartial<BaseNode>) {
+        super(input);
+    }
+
+    @Field(() => GID)
+    id: ID;
+
+    @Field()
+    createdAt: Date;
+
+    @Field()
+    updatedAt: Date;
+}
```

Comment:
BaseNode is exactly the same as VendureEntity with only difference that it defines GraphQL fields.